packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -251,8 +251,8 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
       serviceType,
       status,
       unit,
-      widgetChartType: widget.chart_type,
-      widgetColor: widget.color,
+      widgetChartType: widget.chart_type || 'area',
```

Comment:
May be `widgetChartType: widget.chart_type ?? 'area',
widgetColor: widget.color ?? 'default',`. if our intention is to handle only undefined or null cases?